the-collab-lab/tcl-44-smart-shopping-list
# Style background and final touches

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description
This PR adds the final styling touches to the app.
- We added background images.
- We used the `@apply` directive inside the `.section-radius` to style the three pages' sections.
- We added a yellow background color to the `app.js` not to have white spaces behind the rounded borders.
- We added the utility `flex-1` in order for the section to take the remaining height.



## Related Issue
closes #13 


## Acceptance Criteria

- [x] the app to look professional and welcoming



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    ✓ | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

![final](https://user-images.githubusercontent.com/85396770/170825305-5966ea3e-08ab-41bc-8fed-f9e75e5bca60.gif)




## Testing Steps / QA Criteria
- From your terminal, pull down this branch with `git pull origin rb-jm-dd-style-background` and check that branch out with `git checkout rb-jm-dd-style-background`
- run npm start
- Enjoy the final product :)
